scripts/modal.js

```diff
@@ -0,0 +1,164 @@
+    projectbootstrap: 'bootstrap',
+    projectRuby: 'ruby',
+    projectButton: 'See Project',
+  },
+  {
+    id: 'sixthProject',
+    title: 'Multi Post Stories',
+    closebtn: '<i class="fa fa-times" aria-hidden="true"></i>',
+    html: 'html',
+    bootstrap: 'bootstrap',
+    ruby: 'Ruby On Rails',
+    projectImage: './images/mobilemodal.png',
+    projectDesc: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam amet animi voluptate doloribus at quaerat, vitae dicta sunt possimus voluptatum, facilis vel? Eligendi, harum ipsum excepturi sint at quasi? Repellendus eveniet suscipit quibusdam inventore architecto minus eligendi adipisci. Harum atque voluptate dolore fugiat pariatur eius ullam architecto porro aspernatur nihil.',
+    projectbtnlive: 'https://telanet.github.io/My-Portfolio/',
+    projectbtnsource: 'https://telanet.github.io/My-Portfolio/',
+  },
+];
+
+function modalpopup() {
+  popUp.style.display = 'block';
+  body.appendChild(popUp);
+  const showing = `
+  <div class="popupcont">
+      <div class="popupcontheader">
+        <h2 class="desktophide">${projectList[projectList.length - 1].title}</h2>
+        <h2 class="mobilehide">Keeping track of hundreds  of components website</h2>
+        <button id="closeModal" class="closeModal">${projectList[projectList.length - 1].closebtn}</button>
+      </div>
+      <div class="popupcontlangs">
+        <ul>
+          <li>${projectList[projectList.length - 1].html}</li>
+          <li>${projectList[projectList.length - 1].bootstrap}</li>
+          <li>${projectList[projectList.length - 1].ruby}</li>
+        </ul>
+      </div>
+      <div class="popupcontbody">
+        <div class="popupcontbodyleft">
+          <img src="${projectList[projectList.length - 1].projectImage}" alt="image">
+        </div>
+        <div class="popupcontbodyright">
+          <p class="desktophide">${projectList[projectList.length - 1].projectDesc}</p>
+          <p class="mobilehide">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.</p>
+          <div class="popupcontbodyrightbtns">
+          <a href="https://telanet.github.io/My-Portfolio/"><button type="button" id="modalbtns live">See Live <img src="./images/viewlive.png" alt="view live"></button></a>
+          <a href="https://github.com/telanet/My-Portfolio"><button type="button" id="modalbtns live">See Source <img src="./images/viewsource.png" alt=""></button></a>
+          </div>
+        </div>
+      </div>
+    </div>
+  `;
+  popUp.innerHTML = showing;
+  body.appendChild(popUp);
```

Comment:
- [ ] Kindly fix the issue of the buttons getting inside the paragraph at different screen heights... On my laptop, I see the buttons above the text 👍

![Animation](https://user-images.githubusercontent.com/50721479/195835077-568454f7-65e4-4089-8753-30e4df70534d.gif)

Reply:
Implemented by using calc.